templates/iscsi_dev.j2

```diff
@@ -0,0 +1,4 @@
+{{ ansible_managed }}
+
+This file indicates that {{ vm.host }}@{{ vm.org }} requires an iscsi device to be functional.
+systemd will check for this file is a vm is started. See »/lib/systemd/system/vm_iscsi@.service« for future reference.
```

Comment:
Is this file actually used, besides being copied? I cannot find any other reference to it.

Reply:
Yes it is, systemd checks for this file to determinate iSCSI dependency.